### PR TITLE
Add ClientID SQL Detection

### PR DESCRIPTION
**Pull request by Zhaars**
Added security control page about client-id-sql-detection from AcraEE. I added some basic info about ClientID SQL detection/ignoring. @vixentael please add some general security info if I missed something.

### CODE REVIEW

File: content/acra/security-controls/client-id-sql-detection/_index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: ClientID SQL Detection
+weight: 12
+---
+
+## ClientID Detection from SQL
+
+{{< hint info >}}
+This feature is available in [Acra Enterprise Edition](/acra/enterprise-edition/) only.
```
Comment (vixentael) [inline content/acra/security-controls/client-id-sql-detection/_index.md:9]:
```suggestion
This feature is available in [Acra Enterprise Edition](/acra/enterprise-edition/) only starting from 0.93.
```

or 0.94?

File: content/acra/security-controls/client-id-sql-detection/_index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+{{< /hint>}}
+
+AcraServer supports the ability to retrieve the ClientID from SQL in the runtime dynamically.
+
+AcraServer uses a default behavior in which the ClientID is retrieved from TLS
+certificates [`--tls_client_id_from_cert` flag]({{< ref "acra/configuring-maintaining/general-configuration/acra-server/#tls" >}}), provided by the client during handshake.
```
Comment (vixentael) [inline content/acra/security-controls/client-id-sql-detection/_index.md:15]: you know, that you can write simply `[--tls_client_id_from_cert flag](/acra/configuring-maintaining/general-configuration/acra-server/#tls)`, right? :)

File: content/acra/security-controls/client-id-sql-detection/_index.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Example of MySQL SET command (`--client_id_sql_varname='@app.client_id'`):
+
+```
+set @app.client_id TO 'client_id'
+```
+
+## ClientID Ignoring
+
+{{< hint info >}}
+This feature is available in [Acra Enterprise Edition](/acra/enterprise-edition/) only.
```
Comment (vixentael) [inline content/acra/security-controls/client-id-sql-detection/_index.md:40]:
```suggestion
This feature is available in [Acra Enterprise Edition](/acra/enterprise-edition/) only starting from 0.93.
```

or 0.94?

File: content/acra/security-controls/client-id-sql-detection/_index.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+{{< /hint>}}
+
+AcraServer also supports the ability to ignore and skip encryption/decryption for specified ClientIDs. TLS-extracted, SQL detected ClientIDs are applicable to ignore by AcraServer.
```
Comment (vixentael) [inline content/acra/security-controls/client-id-sql-detection/_index.md:43]:
```suggestion
AcraServer supports the ability to ignore and skip encryption/decryption for specified ClientIDs. If clientID is extracted from TLS certificate, AcraServer can match it with "Ignored ClientIDs" like and do NOT perform encryption/decryption for this client, send its queries "as is".
```

File: content/acra/security-controls/client-id-sql-detection/_index.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+```
+set @app.client_id TO 'client_id'
+```
+
+## ClientID Ignoring
+
+{{< hint info >}}
+This feature is available in [Acra Enterprise Edition](/acra/enterprise-edition/) only.
+{{< /hint>}}
+
```
Comment (vixentael) [inline content/acra/security-controls/client-id-sql-detection/_index.md:42]:
```suggestion

Sometimes it's useful NOT to add encryption/decryption for certain applications ("clients" in Acra terms). Now you can configure a list of "Ignored ClientIDs": a traffic coming from them won't be parsed, encrypted or decrypted.
```

File: content/acra/security-controls/client-id-sql-detection/_index.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+title: ClientID SQL Detection
+weight: 12
+---
+
+## ClientID Detection from SQL
+
+{{< hint info >}}
+This feature is available in [Acra Enterprise Edition](/acra/enterprise-edition/) only.
+{{< /hint>}}
+
```
Comment (vixentael) [inline content/acra/security-controls/client-id-sql-detection/_index.md:11]: we should add 1-2 sentence on why this feature is needed and why users will want to use it. I.e. some use cases would be useful.

File: content/acra/security-controls/client-id-sql-detection/_index.md (hunk 2 above)
Comment (vixentael) [inline content/acra/security-controls/client-id-sql-detection/_index.md:15]:
```suggestion
AcraServer encrypts/decrypts data per client (application, user or group of users). To distinguish the clients, AcraServer uses "ClientID" which it retrieves from TLS certificate provided by the client during handshake (use flag [`--tls_client_id_from_cert`](/acra/configuring-maintaining/general-configuration/acra-server/#tls)).
```

**Review by vixentael — COMMENTED** [review]
great draft, let's add more clarity!

**Review by vixentael — APPROVED** [review]

**Review by Lagovas — APPROVED** [review]
awesome

**Pull request merged**